.travis.yml

```diff
@@ -1,13 +1,20 @@
 language: python
-python: 2.7
+python: 
+    - 2.6
+    - 2.7
+    - 3.3
+    - 3.4
+    - pypy
 env:
 - TOXENV=py26
 - TOXENV=py27
 - TOXENV=py33
 - TOXENV=py34
+- TOXENV=pypy
 install:
 - pip install tox
 - pip install -U cython
+- pip install pypy
```

Comment:
Oh god, how did I make that mistake. I must have been a bit exhausted when I did this commit.